src/modules/75-cf/pages/environment-details/__tests__/EnvironmentDetailsBody.test.tsx

```diff
@@ -130,4 +132,71 @@ describe('EnvironmentDetailsBody', () => {
 
     expect(screen.queryByText('cf.environments.apiKeys.redactionWarning')).not.toBeInTheDocument()
   })
+
+  test('it should show Delete SDK key confirmation modal when trash icon is clicked', async () => {
+    const mutate = jest.fn(() => {
+      return Promise.resolve({ data: {} })
+    })
```

Comment:
You can simplify this with `mockResolvedValue`
```suggestion
    const mutate = jest.fn().mockResolvedValue({ data: {} })
```